### ISSUE TITLE
Does not complile

### ISSUE THREAD

**Issue by zaczh** [zulip/zulip-ios#4]
The project does not compile with XCode 7.

**Comment by lfranchi** [comment]
This is true. IIRC this is due to our DTCoreText submodule, which we just switched to CocoaPods. I think switching to grabbing DTCoreText via cocoapods (`pod install`) plus then opening `Zulip.xcworkspace`  could be enough.

**Comment by knowcliu** [comment]
Here are the steps Jasdev and I took to resolve these issues:

Add this to the Podfile:
`pod 'DTCoreText'`

Run
```
pod install
git submodule update --init --recursive

```

Open the workspace file in XCode
Resolve the ! Errors by clicking on them and go through the "Convert to XCTest" wizard thinggy and click Cancel if it doesn't let you do anything else ... and magic... all the errors are gone-ish.
Update XCode Recommended settings

Remove the DTCoreText section from the `.gitmodules` file
Also remove the physical folder: `rm -rf ThirdParty/DTCoreText/`
If it's still in the XCode project, delete it there too: ThirdParty/DTCoreText

Clean all the things: Cmd + Option + shift + K
Remove the Zulip > iPhone target > Build Phases ... Run Script Phase that sounds like Crashlytics target

**Comment by karamcnair** [comment]
For those not so familiar with Xcode (yet), please note that by default sometimes of us open the project file instead of the workspace file. 

Zulip.xcodeproj is NOT what you want.
Zulip.xcworkspace IS what you want :)
